---
title: Public Data
authors:
- zachary_brehm
- isar_nassiri
- admin
summary: Leveraging public data resources.
tags:
- public_data
date: "2016-04-27T00:00:00Z"

# Optional external URL for project (replaces project detail page).
external_link: ""

image:
  caption: 
  focal_point: 

links:
#- icon: twitter
#  icon_pack: fab
#  name: Follow
#  url: https://twitter.com/georgecushen
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: example
---

Dr. McCall's early research focused on developing statistical methods that made use of the plethora of publicly available data to preprocess and analyze data from a single microarray. The methodology we developed, frozen robust multi-array analysis (fRMA) and the gene expression barcode, permitted researchers to obtain robust measurements of absolute expression from individual samples. Additionally, these methods reduced batch- and platform-specific effects, thereby facilitating the integration of data from different studies and technologies. 

The McCall Lab has continued to develop and apply methods to leverage publicly available data. In collaboration with Dr. Halushka, we combined 8 billion small RNA-seq reads from 46 primary cell types, 42 cancer or immortalized cell lines, and 26 tissues to establish the landscape of human cell-specific microRNA expression. Most recently, we used data from the Library of Integrated Network-based Cellular Signatures (LINCS) project to investigate the dependence between changes in cell morphology and gene expression in response to small molecular perturbations. 
